README.md

```diff
@@ -57,9 +60,33 @@ unless real values don't help users know what to change.
 
 ```hcl
 
+module "watsonx_assistant" {
+  source            = "terraform-ibm-modules/watsonx-assistant/ibm"
+  region            = "us-south"
+  assistant_name    = "example-wx-assistant-basic"
+  plan              = "lite"
```

Comment:
`lite` is not one of the plans you have listed in the allowed values?

Reply:
corrected. Oversight.